api/envoy/config/bootstrap/v2/bootstrap.proto

```diff
@@ -245,14 +246,17 @@ message RuntimeLayer {
 
   // :ref:`Disk runtime <config_runtime_local_disk>` layer.
   message DiskLayer {
-    // Specifies the subdirectory to load within the :ref:`root directory
-    // <envoy_api_field_config.bootstrap.v2.LayeredRuntime.symlink_root>`. This
-    // is useful if multiple systems share the same delivery mechanism. Envoy
-    // configuration elements can be contained in a dedicated subdirectory.
-    string subdirectory = 1;
```

Comment:
Thought: If we wanted to allow uses to optimize the overlapping reloads, could we just keep subdirectory? That would allow the same root, but different subdirectories? I don't feel strongly about this but just throwing it out there.

Reply:
I think that if we want to optimize reloads, we can make them more specific, i.e. only reload from the subtree for the disk watch in question, rather than all disk watches. If the trees are disjoint, then what we have is fine. If they are nested, then we probably just want to set the watch on the outer symlink root in an optimized setting.